driver/controller.go

```diff
@@ -950,34 +951,35 @@ func (d *Driver) waitAction(ctx context.Context, volumeId string, actionId int)
 		"action_id": actionId,
 	})
 
-	ctx, cancel := context.WithTimeout(ctx, time.Minute)
+	// This timeout should not strike given all sidecars use a timeout that is
+	// lower (which should always be the case). Using this as a fail-safe
+	// mechanism in case of a bug or misconfiguration.
+	ctx, cancel := context.WithTimeout(ctx, d.waitActionTimeout)
 	defer cancel()
 
-	// TODO(arslan): use backoff in the future
```

Comment:
Note to reviewers: I did not carry this TODO intentionally since backoffs for timeouts are implemented by the sidecars, and other kinds of errors we occasionally run into do not seem to warrant an exponential backoff as of today (e.g., temporary 4xx responses returned from the API that disappear on the next API call).